packages/manager/modules/dedicated-universe-components/CHANGELOG.md

```diff
@@ -0,0 +1,419 @@
+## [2.0.3](https://github.com/ovh/manager/compare/@ovh-ux/ng-ovh-cloud-universe-components@2.0.2...@ovh-ux/ng-ovh-cloud-universe-components@2.0.3) (2020-08-18)
```

Comment:
Changelog should be empty if it's a new module. Looks like it was copy/pasted from `cloud-universe-components`. Any issue with using the sao module generator ?